.github/workflows/deploy.yml

```diff
@@ -0,0 +1,89 @@
+# ref: https://packaging.python.org/en/latest/guides/publishing-package-distribution-releases-using-github-actions-ci-cd-workflows
+
+name: Publish Python 🐍 distribution 📦 to PyPI and TestPyPI
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  build:
+    name: Build distribution 📦
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v4
+    - name: Set up Python
+      uses: actions/setup-python@v4
+      with:
+        python-version: "3.x"
+    - name: Install pypa/build
+      run: >-
+        python3 -m
+        pip install
+        wheel sdist setuptools_scm
+        --user
+    - name: Build a binary wheel and a source tarball
+      run: python3 setup.py sdist bdist_wheel
+    - name: Store the distribution packages
+      uses: actions/upload-artifact@v3
+      with:
+        name: python-package-distributions
+        path: dist/
+
+
+# This will be triggered everytime when tag is created
+  publish-to-pypi:
+    name: >-
+      Publish Python 🐍 distribution 📦 to PyPI
+    if: startsWith(github.ref, 'refs/tags/')  # only publish to PyPI on tag pushes
+    needs:
+    - build
+    runs-on: ubuntu-latest
+    environment:
+      name: pypi
+      url: https://pypi.org/p/stock-indicators
+    permissions:
+      id-token: write  # IMPORTANT: mandatory for trusted publishing
+
+    steps:
+    - name: Download all the dists
+      uses: actions/download-artifact@v3
+      with:
+        name: python-package-distributions
+        path: dist/
+    - name: Publish distribution 📦 to PyPI
+      uses: pypa/gh-action-pypi-publish@release/v1
+    - name: Tag and draft release note
+      uses: ncipollo/release-action@v1
+      with:
+        body: "We’ve released a new Stock Indicators for Python.  See [Skender.Stock.Indicators @ PyPI.org](${{ vars.PYPI_DOWNLOAD_PREFIX }}${{ github.ref_name }}) for more information."
+        generateReleaseNotes: true
+        draft: true
```

Comment:
I've added this configuration as you recommended. Please set PYPI_DOWNLOAD_PREFIX to `https://pypi.org/project/stock-indicators/`

Reply:
I've added this to the `pypi` environment, and also an equivalent one for the `pypi-test` URL, but suspect that'd never get used; just added it for parity.

![image](https://github.com/DaveSkender/Stock.Indicators.Python/assets/8432125/ae412f26-cfbc-4d58-9afb-060a1f5599fc)

Reply:
I also updated to the current `deploy.yml` filename on PyPI settings.

![image](https://github.com/DaveSkender/Stock.Indicators.Python/assets/8432125/0874a921-ba7f-4636-a391-4a81217638ea)